components/categories-filter/CategoriesFilter.jsx

```diff
@@ -35,74 +98,150 @@ export default class CategoriesFilter extends Component {
+                  onClick={this.toggleExpand}
+                  onKeyPress={this.toggleExpand}
+                  className={classNames('filter__toggler', {
+                    'filter__toggler--expanded': isExpanded,
+                    'filter__toggler--selected': selectedCategories.length,
+                  })}
+                >
+                  {
+                    selectedCategories.length
+                      ? `${selectedCategories.length} filters selected`
+                      : 'Set the filters'
+                  }
+                </div>
+
+                <SlideDown
+                  closed={!isExpanded}
+                  className="filter__slidedown"
+                >
+                  <ul className="filter__list">
+                    <div
+                      className="filter__categories"
+                    >
+                      {categoriesList.map((category) => (
+                        <CategoryButton
+                          category={category}
+                          key={category}
+                          isActive={selectedCategories.includes(category)}
+                          buttonClick={() => this.selectCategory(category)}
+                        />
+                      ))}
+                    </div>
+                    <div className="filter__controls">
+                      <CategoryButton
+                        category="Clear"
+                        buttonClick={this.clearCategories}
+                        className={classNames('-clear', { '-hidden': !selectedCategories.length })}
+                      />
+                      <CategoryButton
+                        category="Show All"
+                        buttonClick={this.selectAllCategories}
+                        className="-show-all"
+                      />
+                    </div>
+                  </ul>
+                </SlideDown>
+              </div>
+            )
+          }
+
+          {
+            isMobile === false
+            && (
+              <div className="filter__wrapper">
+                <ul className="filter__list">
+                  <div
+                    className="filter__categories"
+                    style={{ left: scroll * -CATEGORY_BUTTON_WIDTH_DESKTOP }}
+                  >
+                    {categoriesList.map((category) => (
+                      <CategoryButton
+                        category={category}
+                        key={category}
+                        isActive={selectedCategories.includes(category)}
+                        buttonClick={() => this.selectCategory(category)}
+                      />
+                    ))}
+                  </div>
+                </ul>
+
+                <div className="filter__arrows">
+                  {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+                  <button
+                    type="button"
+                    className="filter__arrow -left"
```

Comment:
I think it will be better to concat these names `filter__arrow -left -> filter__arrow-left`